pkg/api/types.go

```diff
@@ -14,6 +15,49 @@ const (
 	StatusError        OpenSCAPStatus = "Error"
 )
 
+// ScanResult represents the compacted result of all scans performed on the image
+type ScanResult struct {
+	// ImageName is a full pull spec of the input image
+	ImageName string `json:"imageName"`
+	// ImageIUD is a SHA256 identifier of the scanned image
+	ImageID string `json:"imageID"`
```

Comment:
The ImageName might be `docker.io/foo/bar:latest`.... I guess we also want to make sure that the image we scanned was the exact image we use in the pod... Actually that opens a new problem for the scanning...
Let say I have Pod that has image set to `docker.io/foo/bar:latest`. The image inspector will be ran with "let scan `docker.io/foo/bar:latest` but that image might not be the same as in PodSpec (might be more new...).

@simon3z how CF deals with this?